manifests/init.pp

```diff
@@ -186,7 +186,7 @@
 #
 # $proxy_password::             Proxy password for authentication
 #
-# $yum_max_speed::              The maximum download speed for a Pulp task, such as a sync. (e.g. "4 Kb" or 1Gb")
+# $yum_max_speed::              The maximum download speed for a Pulp task, such as a sync. (e.g. "4 kb" or 1gb")
```

Comment:
to_bytes is weird. `k[bB]` is used, all the others use upper case prefixes: `M[Bb]` `G[Bb]` etc.